### PR TITLE
chore: update webextension-polyfill

### PR DESCRIPTION
**Pull request by dylancom**
#### Type of change (Remove other not matching type)

- New feature (non-breaking change which adds functionality)

#### Describe the changes you have made in this PR -
Upgrade from 0.8.0 to 0.9.0. This breaks our tests initially, so a small change to the setup was needed.

### CODE REVIEW

File: jest.setup.ts [hunk 1]
```diff
@@ -3,3 +3,6 @@
 // expect(element).toHaveTextContent(/react/i)
 // learn more: https://github.com/testing-library/jest-dom
 import "@testing-library/jest-dom";
+
+// fix "This script should only be loaded in a browser extension."
```
Comment (bumi) [inline jest.setup.ts:7]:
```suggestion
// fix "This script should only be loaded in a browser extension." e.g. https://github.com/mozilla/webextension-polyfill/issues/218
```

**Review by bumi — APPROVED** [review]

**Comment by escapedcat** [comment]
Funny, prepared that here as well :P  
https://github.com/getAlby/lightning-browser-extension/pull/768

**Comment by bumi** [comment]
oh I also did not realize there was an update in the depfu PR. sorry.

**Comment by dylancom** [comment]
I was looking for a way to avoid adding extra tsconfig rules (by adding "@types/chrome").
But should have committed here actually, sorry

**Pull request merged**